js/runner.go

```diff
@@ -122,6 +124,42 @@ func (r *Runner) NewVU(samplesOut chan<- stats.SampleContainer) (lib.VU, error)
 	return lib.VU(vu), nil
 }
 
+func getRandomIP(ifacesList string) (*net.IPAddr, error) {
+	var addresses []net.Addr
+
+	for _, ifaceName := range strings.Split(ifacesList, ",") {
+		targetIf, err := net.InterfaceByName(ifaceName)
+		if err != nil {
+			return nil, errors.New("Cannot find network interface '" + ifaceName + "'")
+		}
+
+		addrs, err := targetIf.Addrs()
+		if err != nil {
+			return nil, errors.New("Cannot get addresses on network interface '" + ifaceName + "'")
+		}
+
+		addresses = append(addresses, addrs...)
+	}
```

Comment:
This code should probably be called only one time, it doesn't make sense to execute it for every VU initialization. For now a [`sync.Once()`](https://golang.org/pkg/sync/#Once) should do, though long-term (i.e. after most of #883 is resolved), we can move most of this logic in a proper config validation layer and have the list of all possible IPs _before_ we even start doing any VU initialization.